roles/gitops_operations/README.md

```diff
@@ -0,0 +1,35 @@
+gitops_operations
+=================
+
+This Ansible role contains varios tasks for GitOps operations.
+
+Requirements
+------------
+
+* Access to an OpenShift cluster
+* Properly configured kubeconfig file for cluster access
+
+Role Variables
+--------------
+
+* `gops_kubeconfig_file`: (string) The file path of kubeconfig file for cluster. Default: undefined, If not provided, and no other connection options are provided, the Kubernetes client will attempt to load the default configuration file from ~/.kube/config. Can also be specified via `K8S_AUTH_KUBECONFIG` environment variable.
+* `gops_app_name`: (string) The name of Application in GitOps.
+* `gops_namespace`: (string) The namespace to use.
+
+Example Playbook
```

Comment:
Rather than "Example Playbook", you may have a chapter here to describe the different operations that are included in this role. E.g. sync_repo, since it's the only one currently available